### ISSUE TITLE
Improve Toolbox Icon Colors for Dark Themes

### ISSUE THREAD

**Issue by Bella-Roma** [powertab/powertabeditor#411]
**Description**

I'm currently running the Alpha 19 build (compiled from source) on Linux Mint 20.2. 

The toolbox and menu icons are very difficult to read with my system's dark theme, as the tool box area is dark gray while the icons are black.  Adding an option under Preferences/Interface to change the color of the icons that appear in the toolbox to compensate for dark themes would be great.

Thanks!

**Comment by cameronwhite** [comment]
yeah, this would be a nice improvement. On the macOS dark theme the icons are somewhat legible, but proper icons would definitely help.
For future testing, which dark theme are you using on Linux Mint?

**Comment by Bella-Roma** [comment]
Hi-

Mint comes with a number of dark options, the _Mint-Y-Dark_ themes, which all share the same "dark" elements, but have different folder/accent colors.

I also tried the _Adwaita-Dark_ theme.

As a work around, I used _QT5 Settings_ to change QT app themes and settled on "Fusion" in the appearance tab with the _Default_ palette, which is actually a light theme. There were a few darker options, but I found the default QT5 palette option to offer the best contrast of text/icons against background colors.
